src/sdb.c

```diff
@@ -449,7 +449,7 @@ static int sdb_set_internal (Sdb* s, const char *key, char *val, int owned, ut32
 			if (cas && kv->cas != cas) {
 				return 0;
 			}
-			if (vlen == kv->value_len && !strcmp (kv->value, val)) {
+			if (vlen == kv->value_len && strcmp (kv->value, val) != 0) {
```

Comment:
Thanks for your feedback, would this mean that the tests in cas.c (which appear to have also failed [in the last successful build on the CI](https://travis-ci.org/radare/sdb/builds/152275982#L279)) are incorrect, not the code itself?